index.js

```diff
@@ -1,17 +1,51 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {4
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length++; 
+  }
 
-  get(pos) {}
+  get(pos) {
+    if(pos>this.items.length) throw new Error('OutOfBounds');
+    return this.items[pos];
+  }
 
-  max() {}
+  max() {
+    if(!this.items.length) throw new Error('EmptySortedList');
+    let maxValue = 0;
```

Comment:
if the array is sorted, then you don't need to check every item for the biggest number; you know where it is